azure-core/azure/core/exceptions.py

```diff
@@ -117,8 +118,8 @@ def __init__(self, message=None, response=None, **kwargs):
                         self.error.error.code,
                         self.error.error.message)
             except AttributeError:
-                if self.error.message:
-                    message = self.error.message
+                if self.error.message: #pylint: disable=no-member
```

Comment:
hmm you may need to rebase - I think most of this `self.error` parsing has been removed from the latest code since we merged changes to the error hierarchy?

Reply:
https://github.com/Azure/azure-sdk-for-python/blame/azure-core/azure-core/azure/core/exceptions.py#L120
I was hoping this changed after I rebased it yesterday..but it didn't. :(